superset/assets/src/SqlLab/components/TableElement.jsx

```diff
@@ -190,7 +192,11 @@ class TableElement extends React.PureComponent {
         </div>
         <div className="pull-right">
           {table.isMetadataLoading || table.isExtraMetadataLoading ?
-            <Loading size={50} />
+            <Loading
+              size={50}
+              position="normal"
+              styleOverrides={STYLE_ZERO_MARGIN}
```

Comment:
Can we do this by defining `margin: 0` in a ccs class and add that css class to the `Loading` component?